packages/core/state/selection/reducer.ts

```diff
@@ -123,9 +123,9 @@ export default makeReducer<SelectionStateBranch>(
             sortColumn: action.payload,
         }),
         [DESELECT_DISPLAY_ANNOTATION]: (state, action) => {
-            const displayAnnotations = without(
```

Comment:
this was the source of the reselect bug I was talking about, for some reason the annotations returned by the persist are slightly different and this dont get removed with the `without` function. The same is true for comparing the objects directly. This is what i was saying about not liking my solution as much.

Reply:
Looks like lodash's `without` function doesn't support this kind of object comparison -- unless the objects share a reference, they aren't considered equal. I'm curious if this use of `without` worked when the payload to `deselectDisplayAnnotation` was previously just `annotation` instead of `{annotation}`?
But also am fine with the use of `filter` here